actioncable/lib/assets/javascripts/action_cable/connection.coffee

```diff
@@ -75,7 +79,17 @@ class ActionCable.Connection
     error: ->
       @disconnect()
 
```

Comment:
The `events` object here is for event handlers. Move `dispatchEvent` to the root of the class and call `@dispatchEvent`, not `@events.dispatchEvent`.